ui/desktop/src/ChatWindow.tsx

```diff
@@ -114,6 +114,25 @@ function ChatContent({
     },
   });
 
+  // Add keyboard shortcut handler for stopping
+  useEffect(() => {
+    const handleKeyDown = (event: KeyboardEvent) => {
+      // Stop on Control+C
+      if (event.ctrlKey && event.key === 'c') {
+        event.preventDefault(); // Prevent the default copy behavior
+        stop();
+      }
```

Comment:
@Kvadratni hrm yes... on CLI muscle memory tells us to use control C - but it is also copy - it would be too risky to make it do only it when text doesn't have focus I think?

Reply:
From the ux perspective we need to think about in app muscle memory

Reply:
agreed - not sure what I would want in here - I think a click like we have? am wary of adding a hotkey at the moment unless enough people reach for it intuitively?